tools/jdk/local_java_repository.bzl

```diff
@@ -147,10 +147,15 @@ def _local_java_repository_impl(repository_ctx):
 
     # Prepare BUILD file using "local_java_runtime" macro
     build_file = ""
+    if repository_ctx.attr.build_file_content:
+        build_file = repository_ctx.attr.build_file_content
     if repository_ctx.attr.build_file != None:
+        if build_file != "":
+            fail("build_file and build_file_content are exclusive")
```

Comment:
nit: I think it improves readability to have this validation at the top level instead of being nested inside conditionals. Something like
```
if repository_ctx.attr.build_file_content and repository_ctx.attr.build_file:
    fail()

if repository_ctx.attr.build_file_content:
     ....
elif repository_ctx.attr.build_file:
     ....
```